api/report_server.thrift

```diff
@@ -170,18 +177,18 @@ service codeCheckerDBAccess {
                        1: i64 reportId)
                        throws (1: shared.RequestFailed requestError),
 
-  // get the results for one runId
+  // get the results for some runIds
   ReportDataList getRunResults(
-                               1: i64 runId,
+                               1: list<i64> runIds,
```

Comment:
Optional means &mdash; I think &mdash; that the `null` value is not propagated over the wire (and on Python's end, `None` is filled).

How will this change filtering? Sending more `ReportFilter` objects, IIRC, currently made them come together in an `OR` clause, while the values in the **same** `ReportFilter` instance manifested as an `AND` clause. This way, we will only be able to express `OR` between the same "dimensions" of data, and no `AND` between filters?